tests/WE2E/test_configs/wflow_features/config.get_from_HPSS_ics_GDAS_lbcs_GDAS_fmt_netcdf_2022040400_ensemble_2mems.yaml

```diff
@@ -1,14 +1,15 @@
 metadata:
   description: |-
     This test checks the capability of the workflow to retrieve from NOAA
-    HPSS netcdf-formatted output files generated by GDAS.
+    HPSS netcdf-formatted output files generated by GDAS. Also ensures that
+    LSM SPP settings can be activated properly.
 user:
   RUN_ENVIR: community
 platform:
   EXTRN_MDL_DATA_STORES: hpss
 workflow:
   CCPP_PHYS_SUITE: FV3_HRRR
-  PREDEF_GRID_NAME: RRFS_CONUS_3km
+  PREDEF_GRID_NAME: RRFS_CONUS_13km
```

Comment:
For future reference: this reduced the core hours used by at least an order of magnitude